components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -1184,6 +1184,12 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
             const project = CommitContext.is(context)
                 ? await this.projectDB.findProjectByCloneUrl(context.repository.cloneUrl)
                 : undefined;
+
+            //TODO(se) relying on the attribution mechanism is a temporary work around. We will go to explicit passing of organization IDs soon.
+            const attributionId = await this.userService.getWorkspaceUsageAttributionId(user, project?.id);
+            const organization =
+                attributionId.kind === "team" ? await this.teamDB.findTeamById(attributionId.teamId) : undefined;
```

Comment:
We only need to pass the ID into the subsequent call, so we can avoid this lookup.